runner.py

```diff
@@ -296,6 +296,29 @@ def validate_input(output_dir, input_dir, reference_file, mode):
         raise Exception("Could not find files ending with '.fastq' or 'fastq.gz' in input_dir !")
 
 
+def create_stats_file(output_dir, filenames, overlapping_reads, reads_input_num):
+    file_path = os.path.join(output_dir, "stats_file.stats")
```

Comment:
please use meaningful variable names - which file's path is this referring to?
also, the convention used in this application is that the suffix of the file represents the type of file so stats.txt would be more meaningful and not redundant..

Reply:
changed